src/reactionsystem.jl

```diff
@@ -27,24 +21,12 @@ function ModelingToolkit.ODESystem(model::SBML.Model; kwargs...)
     convert(ODESystem, rs, defaults=defaults)
 end
 
-""" ODESystem constructor """
-function ModelingToolkit.ODESystem(sbmlfile::String; kwargs...)
-    model = readSBML(sbmlfile)
-    ODESystem(model; kwargs...)
-end
-
 """ ODEProblem constructor """
 function ModelingToolkit.ODEProblem(model::SBML.Model,tspan;kwargs...)  # PL: Todo: add u0 and parameters argument
```

Comment:
This one I think we want to remove too, right @ChrisRackauckas

Reply:
yeah it's a bit odd since no other repo goes right to numerical. This wouldn't give the possibility of running `structural_simplify`